docs/components/popover.md

```diff
@@ -1,760 +1,515 @@
+    <KPop
+      button-text="Open popover"
+      :hide-close-icon="data.hideCloseIcon"
+    >
+      <template #content>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      </template>
+    </KPop>
+  </div>
+</KComponent>
 
 ```html
-<KPop title="Cool header" width="300">
-  <KButton>button</KButton>
+<KPop
+  hide-close-icon
+  button-text="Open popover"
+>
   <template #content>
-    I am 300 pixels wide!
+    Popover content.
   </template>
 </KPop>
 ```
 
-### maxWidth
+### width
 
-The max width of the popover body - by default it is `auto`.
+Width of the popover container. Default value is `200px`.
 
-<KPop title="Cool header" width="auto" max-width="500">
-  <KButton>button</KButton>
+<KPop
+  width="500"
+  button-text="Open popover"
+>
   <template #content>
-    I am 500 pixels wide! I am 500 pixels wide! I am 500 pixels wide! I am 500 pixels wide! I am 500 pixels wide!
+    Popover content.
   </template>
 </KPop>
 
 ```html
-<KPop title="Cool header" max-width="500">
-  <KButton>button</KButton>
+<KPop
+  width="500"
+  button-text="Open popover"
+>
   <template #content>
-    I am 500 pixels wide! I am 500 pixels wide! I am 500 pixels wide! I am 500 pixels wide! I am 500 pixels wide!
+    Popover content.
   </template>
 </KPop>
 ```
 
-### zIndex
+### maxWidth
 
-The `z-index` of the popover - by default it is `1000`.
+Maximum width of the popover container. Default value is `auto`.
 
-```html
-<KPop title="Cool header" z-index="9999">
-  <KButton>button</KButton>
-  <template v-slot:content>
-    I have a z-index of 9999! I have a z-index of 9999!
+<KPop
+  max-width="120"
+  button-text="Open popover"
+>
+  <template #content>
+    Popover content.
   </template>
 </KPop>
-```
-
-### popoverClasses
-
-Custom classes that you want to append to the popover - by default it has a `k-popover` class on it.
 
 ```html
-<KPop title="Cool header" popoverClasses="my-class">
-  <KButton>button</KButton>
+<KPop
+  max-width="120"
+  button-text="Open popover"
+>
   <template #content>
-    I have a custom my-class class on me!
+    Popover content.
   </template>
 </KPop>
 ```
 
-### popoverTransitions
+### target
 
-Custom transitions that you want the popover to have - by default it uses a `fade` transition.
+This is the target element that the popover is appended to. By default its KPop wrapper element.
 
-```html
-<KPop title="Cool header" popoverTransitions="slide">
-  <KButton>button</KButton>
-  <template #content>
-    I use a slide transition!
-  </template>
-</KPop>
-```
+### tag
 
-### popoverTimeout
+KPop wrapper element. Default value is `div`.
 
-Custom timeout setting that you want the popover to have - by default it is set to 300 milliseconds.
+### popoverClasses
 
-<KPop title="Cool header" :popover-timeout="1000" trigger="hover">
-  <KButton>button</KButton>
-  <template #content>
-    I have a 1 second timeout!
-  </template>
-</KPop>
+List of class names you want to assign to `.k-popover` element.
```

Comment:
is it still `k-popover` or did we remove the `k-`?

Reply:
It's still `k-popover`